impl/maven-cli/src/main/java/org/apache/maven/cling/logging/impl/MavenSimpleConfiguration.java

```diff
@@ -37,7 +41,14 @@ public void setRootLoggerLevel(Level level) {
                     case INFO -> "info";
                     default -> "error";
                 };
-        System.setProperty("org.slf4j.simpleLogger.defaultLogLevel", value);
+
+        String current = System.getProperty(MavenSimpleLogger.DEFAULT_LOG_LEVEL_KEY);
```

Comment:
```suggestion
        String current = System.setProperty(MavenSimpleLogger.DEFAULT_LOG_LEVEL_KEY, value);
        if (current != null && !Objects.equals(value, current)) {
```

Reply:
Does this logically seem to be maven command options taking precedence over slf4j config?
@gnodet

Reply:
Yes, a CLI option seems more explicit to me that a value from system properties.  CLI options are usually set by the user for a given invocation, while system properties can come from various sources (also from CLI, that's right).
I think this would be in line with the example I pasted a few days ago:
```
        String styleColor = mavenOptions
                .color()
                .orElse(userProperties.getOrDefault(
                        Constants.MAVEN_STYLE_COLOR_PROPERTY, userProperties.getOrDefault("style.color", "auto")));
```
where the color option comes first from the CLI options, then user properties (in that case, we have the new, then the legacy property).

Reply:
The log msg also needs to be adjusted.